packages/server/src/orchestrator/atom.ts

```diff
@@ -7,6 +7,11 @@ interface OrchestratorAtomOptions {
 
 export const createOrchestratorAtom = (options?: OrchestratorAtomOptions) => {
   let atom = new Atom<OrchestratorState>({
+    manifestGenerator: {
+      id: '@bigtest/manifest-genertor',
```

Comment:
I find it a bit confusing to name this as though it were a package, when it doesn't exist as a package.

Reply:
I have removed the `id` and `name` fields until we actually know what their use is.